db/migrate/20220125155705_remove_mobile_vaccine_group_name.rb

```diff
@@ -0,0 +1,5 @@
+class RemoveMobileVaccineGroupName < ActiveRecord::Migration[6.1]
+  def change
+    safety_assured {  remove_column :mobile_vaccines, :group_name }
```

Comment:
Added migration but omitted the updated schema file. That'll be generated when they run migration in prod.

Reply:
🚨 Note that migrations do not run automatically, you'll need to open a #vfs-platform-support ticket via slack when you merge this in 🚨

Reply:
My memory here may be backwards here, but I think @kreek told me to put migrations in the mobile module. If so, I obviously messed it up (probably while breaking my PR into several smaller PRs) and put it in rails root. It might be best to leave this one here for consistency, but it would be good to get a reminder of where mobile migrations should go.

Reply:
yes, good catch @kpethtel, we have our own migrations for mobile and this file should move there.

Reply:
Not a biggie - PRs are a team sport 😄 - the reason for putting migrations in our module is that we have code ownership over it and don't need VFS approval to merge it in.

Reply:
I've moved the migration over to the mobile module but Github is giving me the error 'Modified files in db/ should be the only files checked into this PR.'. Should I breakout the migration into it's own PR?

Reply:
> I've moved the migration over to the mobile module but Github is giving me the error 'Modified files in db/ should be the only files checked into this PR.'. Should I breakout the migration into it's own PR?

Yes, we can merge them back to back.

Reply:
Changed the migration to be in mobile module in it's own PR. (https://github.com/department-of-veterans-affairs/vets-api/pull/8969) but running the migration locally it seems like the mobile_vaccines table is out of scope. running `rails db:migrate` gives `PG::UndefinedColumn: ERROR:  column "group_name" of relation "mobile_vaccines" does not exist`. 

It looks like we have a task `mobile:install:migrations` but that just copies the the migrations into the base application and then `db:migrate` gives error due to multiple migrations with the same name.

Reply:
Nevermind... It would probably help if I did a rollback before trying again.